File: smoke-tests-otel-starter/spring-boot-3/src/main/java/io/opentelemetry/spring/smoketest/Customer.java
package io.opentelemetry.spring.smoketest;

import org.springframework.data.annotation.Id;


public class Customer {
	@Id
	public String id;

	public String firstName;
	public String lastName;

	public Customer() {}

	public Customer(String firstName, String lastName) {
		this.firstName = firstName;
		this.lastName = lastName;
	}

  public String getFirstName() {
    return firstName;
  }

  public String getId() {
    return id;
  }

  public String getLastName() {
    return lastName;
  }
}

File: instrumentation/spring/spring-data/spring-data-3.0/kotlin-testing/src/test/kotlin/io/opentelemetry/javaagent/instrumentation/spring/data/v3_0/repository/Customer.kt
/*
 * Copyright The OpenTelemetry Authors
 * SPDX-License-Identifier: Apache-2.0
 */

package io.opentelemetry.javaagent.instrumentation.spring.data.v3_0.repository

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Column
import org.springframework.data.relational.core.mapping.Table

@Table("CUSTOMER")
data class Customer(
  @Id @Column("ID") val id: Long,
  @Column("NAME") val name: String,
)
